nntrainer/compiler/tflite_interpreter.cpp

```diff
@@ -367,7 +455,32 @@ buildOperators(const TfOpNodes &nodes, const TfOpIdxMap &map,
     auto &index_map = map.getIndexMap<tflite::BuiltinOperator>();
 
     auto op_code = index_map.getIndex(node.getOpType());
-    auto inputs = variables_to_idx_vector(node.getInputs());
+    std::vector<int> inputs;
+    if (node.isInputNode()) {
+      inputs = variables_to_idx_vector(node.getInputs());
+    } else {
+      /**
+       *  Q) Why find a tensor that shares a buffer with input tensor?
+       *
+       *  A) the tflite needs only one tensor between nodes. Therefore,
+       *basically, output tensors are used for tflite tensor that shares its
+       *buffer with input's
+       **/
+      TfOpNode::Variables input_tensors;
+      for (auto parent_node : getPredNodes(node)) {
```

Comment:
Add just a comment after the offline discussion,
nntrainer uses different tensor names between layers with the same buffer. So, for the tflite export, we need one tensor for each. In order to do it, we are going to use the previous layers' output tensor.